ChangeLog

```diff
@@ -63,7 +63,24 @@ Changes
    * Reduce RAM consumption during session renegotiation by not storing
      the peer CRT chain and session ticket twice.
 
-= mbed TLS 2.16.X branch released XXXX-XX-XX
+= mbed TLS 2.16.6 branch released 2020-04-14
+
```

Comment:
Changes related with mbed TLS 2.16.6 are newer then changes described in 'mbed TLS "baremetal"' section. Shouldn't be added at the top of this file? Is it a convention that only baremetal changes are always at the top of the ChangeLog file?

Reply:
Looking at how it was handled previously - yes. Previous releases are lower, and baremetal is on top of that.